content/docs/2.6/scalers/azure-app-insights.md

```diff
@@ -0,0 +1,120 @@
++++
+title = "Azure Application Insights"
+layout = "scaler"
+availability = "v1.3+"
+maintainer = "Community"
+description = "Scale applications based on Azure Application Insights metrics."
+go_file = "azure_app_insights_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `azure-app-insights` trigger that scales based on an Azure Application Insights metric.
+
+```yaml
+triggers:
+- type: azure-app-insights
+  metadata:
+    metricAggregationTimespan: "0:1"
+    metricAggregationType: avg
+    metricFilter: cloud/roleName eq 'role_name'
+    metricName: "metric_name"
+    metricNamespace: "customMetrics"
+    targetValue: "1"
+    activeDirectoryClientIdFromEnv: CLIENT_ID_ENV_NAME # Optional, can use TriggerAuthentication as well
+    activeDirectoryClientPasswordFromEnv: CLIENT_PASSWORD_ENV_NAME # Optional, can use TriggerAuthentication as well
+    appIdFromEnv: APP_ID # Optional, can use TriggerAuthentication as well
```

Comment:
```suggestion
    applicationInsightsNameFromEnv: APP_ID # Optional, can use TriggerAuthentication as well
```

Do you mean the name or ID? If you mean the ID, is that the ARM URI or?

Reply:
I meant the ID or GUID. I tried to improve the description to make this more clear. Let me know if it helps.

Reply:
Thanks!

In that case I would rename the field to this:
```suggestion
    applicationInsightsIdFromEnv: APP_ID # Optional, can use TriggerAuthentication as well
```

Because app ID is very confusing in the combination with AD auth in my opinion